src/platform/site-wide/va-footer/helpers.jsx

```diff
@@ -20,19 +20,20 @@ export const FOOTER_EVENTS = {
   CRISIS_LINE: 'nav-footer-crisis',
 };
 
-const renderInnerTag = (link, captureEvent) => [
-  link.label ? (
-    <span className="va-footer-link-label">{link.label}</span>
-  ) : null,
-
-  link.href ? (
-    <a href={link.href} onClick={captureEvent} target={link.target}>
-      {link.title}
-    </a>
-  ) : (
-    <span className="va-footer-link-text">{link.title}</span>
-  ),
-];
+const renderInnerTag = (link, captureEvent) => (
+  <>
+    {link.label ? (
+      <span className="va-footer-link-label">{link.label}</span>
+    ) : null}
+    {link.href ? (
+      <a href={link.href} onClick={captureEvent} target={link.target}>
+        {link.title}
+      </a>
+    ) : (
+      <span className="va-footer-link-text">{link.title}</span>
+    )}
+  </>
+);
```

Comment:
Fragments! 😮 🎉

Reply:
Oops, your PR from yesterday didn't get merged? I'm fine with this change being a part of this PR and killing the other PR that had this change 😀

Reply:
@erikphansen , which "other" PR?

Reply:
yeah that's the PR. It makes this same change. so I figured this branch had that work in it as well.

Reply:
Update: the other branch is now Merged into master (the CI build-check passed once I sync'ed the branch up with master 15-20 min ago and pushed that up)...
So yes, the change is here now too 'cuz I just merged master into it a sec ago.
Just waiting for the check below to pass now.